<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaManagerDialog</class>
 <widget class="QDialog" name="MediaManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Testing Media Renderer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="centralwidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QListWidget" name="itemsList"/>
      </item>
      <item>
       <widget class="QGroupBox" name="verticalGroupBox">
        <property name="title">
         <string>Controls</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="playButton">
           <property name="text">
            <string>&amp;Play</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="text">
            <string>&amp;Stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mediaRenderersCb"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
